code/index.html

```diff
@@ -1,22 +1,96 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+</head>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+<body>
+  <header>
+    <nav>
+      <img class="logo" src="./icons256.png" alt="logo">
+      <h1 class="site-name">NEWS TODAY</h1>
+      <div class="navigation" id="navigation-bar">
+        <ul>
+          <li><a href="default.asp">Log In</a></li>
+          <li><a href="subscribe.asp">Subscribe</a></li>
+          <li><a href="contact.asp">Contact</a></li>
+        </ul>
+    </nav>
+  </header>
+  <hr>
+
+  <section class="big-news">
+    <!-- The big news content -->
+    <img class="big-news-image" src="./paper3.jpeg" alt="big news" />
+    <div class="text-container">
+      <h2 class="big-news-title">BIG NEWS</h2>
+      <p class="big-news-text">
+        Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to
+        emphasise design elements over content.Lorem ipsum is a pseudo-Latin text used in web design, typography,
+        layout, and printing in place of English to
+        emphasise design elements over content.
+      </p>
+    </div>
+  </section>
+
+  <hr>
+  <section class="flex-parent">
+    <div class="flex-child blue">
```

Comment:
Can´t find out what the colour names means in the class flex-child. There is a animation when you come in to the page that ends in white. But I see no yellow. Can you give me a hint? 

Maybe it would be good with a small description on it in the code? If it is not obvious to see on the site.